arch/arc/Kconfig

```diff
@@ -370,6 +370,28 @@ endmenu
 config DCACHE_LINE_SIZE
 	default 32
 
+config DCACHE_REGION_OPERATIONS
+	bool "DCACHE region operations"
+	depends on CACHE_MANAGEMENT && DCACHE && CPU_HS4X
```

Comment:
There are more ARC cores supporting region operations, let's remove the "CPU_HS4X" restriction but make the feature default n.